tools-core/src/main/java/com/hedera/hashgraph/client/core/transactions/ToolTransferTransaction.java

```diff
@@ -147,10 +152,24 @@ public boolean checkInput(JsonObject input) throws HederaClientRuntimeException
 
 	@Override
 	public Set<AccountId> getSigningAccounts() {
+		Map<AccountId, AccountInfo> infos = new HashMap<>();
+		try {
+			infos = loadAccountInfos();
+		} catch (HederaClientException | InvalidProtocolBufferException e) {
+			logger.error(e.getMessage());
```

Comment:
It looks like if we hit this `catch`, we can continue---**but** may omit some necessary signatures for involved accounts with `receiverSigRequired=true`.

So perhaps a `WARN`-level log would be more appropriate, with a message that describes the user impact of the event. E.g.,
```suggestion
			logger.warn("Unable to load account information, some required receiver signatures may be omitted", e);
```

Reply:
Will make the change.